app/controllers/decidim/action_delegator/admin/application_controller.rb

```diff
@@ -6,6 +6,9 @@ module Admin
       # This controller is the abstract class from which all other controllers of
       # this engine inherit.
       class ApplicationController < Decidim::Admin::ApplicationController
+        def permission_class_chain
+          [::Decidim::ActionDelegator::Permissions] + super
+        end
```

Comment:
Perhaps this file could be refactored to follow the usual style for `permission_class_chain`, like in these files:

- https://github.com/decidim/decidim/tree/develop/decidim-initiatives/app/controllers/decidim/initiatives/admin/application_controller.rb
- https://github.com/decidim/decidim/blob/develop/decidim-participatory_processes/app/controllers/decidim/participatory_processes/application_controller.rb

```suggestion
        register_permissions(::Decidim::ActionDelegator::Admin::ApplicationController,
                             ::Decidim::ActionDelegator::Permissions,
                             ::Decidim::Admin::Permissions)
        
        def permission_class_chain
          ::Decidim.permissions_registry.chain_for(::Decidim::ActionDelegator::Admin::ApplicationController)
        end
```